File: php/program/ProdukAudioVisual.php
<?php

require_once 'ProdukElektronik.php';

class ProdukAudioVisual extends ProdukElektronik {
    protected $harga;
    protected $garansi;
    protected $tipe;

    public function __construct($idProduk, $namaProduk, $merk, $gambar, $harga, $garansi, $tipe) {
        parent::__construct($idProduk, $namaProduk, $merk, $gambar); // Meneruskan parameter gambar
        $this->harga = $harga;
        $this->garansi = $garansi;
        $this->tipe = $tipe;
    }

    // Getters
    public function getHarga() { return $this->harga; }
    public function getGaransi() { return $this->garansi; }
    public function getTipe() { return $this->tipe; }

    // Setters
    public function setHarga($harga) { $this->harga = $harga; }
    public function setGaransi($garansi) { $this->garansi = $garansi; }
    public function setTipe($tipe) { $this->tipe = $tipe; }
}

File: java/program/ProdukAudioVisual.java
public class ProdukAudioVisual extends ProdukElektronik {
    private double harga;
    private String garansi;
    private String tipe;

    public ProdukAudioVisual(String idProduk, String namaProduk, String merk, double harga, String garansi, String tipe) {
        super(idProduk, namaProduk, merk);
        this.harga = harga;
        this.garansi = garansi;
        this.tipe = tipe;
    }

    // Getters
    public double getHarga() { return harga; }
    public String getGaransi() { return garansi; }
    public String getTipe() { return tipe; }

    // Setters
    public void setHarga(double harga) { this.harga = harga; }
    public void setGaransi(String garansi) { this.garansi = garansi; }
    public void setTipe(String tipe) { this.tipe = tipe; }
}